src/services/ProjectService.ts

```diff
@@ -164,20 +166,46 @@ export class ProjectService {
   }
 
   private static async createProject(proposal: ProposalWithOutcome) {
-    const newProject: ProjectAttributes = {
+    const creationDate = new Date()
+    const newProject = await ProjectModel.create({
       id: crypto.randomUUID(),
       proposal_id: proposal.id,
       title: proposal.title,
       status: ProjectStatus.Pending,
       links: [],
-      created_at: new Date(),
-    }
+      created_at: creationDate,
+    })
+
+    await ProjectService.createPersonnel(proposal, newProject, creationDate)
 
-    return await ProjectModel.create(newProject)
+    return newProject
+  }
+
+  private static async createPersonnel(
+    proposal: ProposalWithOutcome,
```

Comment:
We don't we pass just `members` or personnel here?

Reply:
I was looking at it as if finding/parsing the info needed to create the personnel was part of this method's responsibility (so `createProject` only handles the project attributes, then delegates the rest)